### PR TITLE
[master] Ignore the fdasd error for rereading partition table

### PR DESCRIPTION
**Pull request by PanYaLian**
The error code of rereading partition table for fdasd command has been changes
it will be 255 in package s390utils-base-2.4.0-1.el7.s390x for rhel7
and 1 in package s390utils-base-2.6.0-28.el8.s390x for rhel8

will ignore the error based on os version

### CODE REVIEW

File: zthin-parts/zthin/bin/unpackdiskimage [hunk 1]
```diff
@@ -1186,7 +1186,22 @@ EOF
...
+        local osRelease="/etc/os-release"
+        if [[ -e $osRelease ]]; then
+            os=`cat $osRelease | grep "^ID=" | sed \
+              -e 's/ID=//' \
+              -e 's/"//g'`
+            version=`cat $osRelease | grep "^VERSION_ID=" | sed \
+              -e 's/VERSION_ID=//' \
+              -e 's/"//g' \
+              -e 's/\.//'`
+            os=$os$version
```
Comment (bjhuangr) [inline zthin-parts/zthin/bin/unpackdiskimage:1199]: why not re-use existing code to get os distro info?

**Comment by bjhuangr** [comment]
And please sign your commit

**Comment by jichenjc** [comment]
it's very weird that they make such changes... anyway, +1 to @bjhuangr 's point above

**Comment by bjhuangr** [comment]
The get os version function also appeared in refreshbootmap and potentially other place. As discussed with Yalian, let's merge this at first, then I will refine this in following PR.

**Pull request merged**